apollo-router/tests/integration/traffic_shaping.rs

```diff
@@ -155,9 +161,12 @@ async fn test_router_timeout_custom_metric() -> Result<(), BoxError> {
     let (_trace_id, response) = router.execute_default_query().await;
     assert_eq!(response.status(), 504);
     let response = response.text().await?;
-    assert!(response.contains("REQUEST_TIMEOUT"));
+    assert!(response.contains("GATEWAY_TIMEOUT"));
```

Comment:
@BrynCooke Fixup these tests to use http.server.response metrics on status.

Reply:
[8ea3123](https://github.com/apollographql/router/pull/6486/commits/8ea31233935e9b5d717d61e07dddf0b191cec2ef)